modin/pandas/dataframe.py

```diff
@@ -257,6 +261,12 @@ def __init__(
         else:
             self._query_compiler = query_compiler
 
+        if query_compiler is None and UsePlainPandasQueryCompiler.get():
+            small_dataframe = pandas.DataFrame(
+                data=data, index=index, columns=columns, dtype=dtype, copy=copy
+            )
+            self._query_compiler = PlainPandasQueryCompiler(small_dataframe)
```

Comment:
```suggestion
```
This can be removed because we already create a qc above by calling from_pandas.

Reply:
done